### DISCUSSION TITLE
Sign in with google provider

### DISCUSSION THREAD

**Discussion by arunkumar413** [nextauthjs/next-auth#10551] [Help]
Hi Team,

I'm trying to sign in using google provider. However I'm getting the below error.  Seems like the issue is related to the redirect URI. I'm testing the app on google's idx editor ( https://idx.google.com/ ) which is a bit similar to codesandbox. It creates its own dev preview URL like this https://9000-monospace-nextjs-routing-1711691269592.cluster-76blnmxvvzdpat4inoxk5tmzik.cloudworkstations.dev/login

Where should I configure the redirect URI in the nextjs app and what should be the redirect URI in this scenario?
```
Access blocked: This app’s request is invalid.

You can’t sign in because this app sent an invalid request. You can try again later or contact the developer about this issue. [Learn more about this error](https://developers.google.com/identity/protocols/oauth2/web-server#authorization-errors-redirect-uri-mismatch)
If you are a developer of this app, see [error details](https://accounts.google.com/).
Error 400: redirect_uri_mismatch
```

Thanks,
Arun

**Comment by tntchn** [comment]
Did you configure right in Google Cloud Platform?
Check 'API & Services' > 'Credentials' > 'OAuth 2.0 Client IDs' and found your web app client. Check that your origins include `https://9000-monospace-nextjs-routing-1711691269592.cluster-76blnmxvvzdpat4inoxk5tmzik.cloudworkstations.dev` and the authorized redirect URIs include `https://9000-monospace-nextjs-routing-1711691269592.cluster-76blnmxvvzdpat4inoxk5tmzik.cloudworkstations.dev/api/auth/callback/google`.

Resources can be found at https://authjs.dev/getting-started/providers/google
